workers/data_refinery_workers/processors/utils.py

```diff
@@ -21,8 +21,7 @@ def start_job(kwargs: Dict):
     job.start_time = timezone.now()
     job.save()
 
-    batch_relations = ProcessorJobsToBatches.objects.filter(processor_job_id=job.id)
-    batches = [br.batch for br in batch_relations]
+    batches = list(job.batches.all())
```

Comment:
Again, this conversion is not really necessary if you only need to get the number of elements in the QuerySet.